src/recipients/RecipientPickerV2.tsx

```diff
@@ -25,6 +26,8 @@ function RecipientPicker({
   isSelectedRecipientLoading,
   style,
 }: Props) {
+  const isRecipientSelected = (recipient: Recipient) =>
```

Comment:
Because of re-renders, it's not guaranteed that we'll have object equality here. The default check is still object equality, but we also now check deep equality as a fallback.